src/runtime/config/configuration-qemu-sev.toml.in

```diff
@@ -32,25 +32,22 @@ machine_type = "@MACHINETYPE@"
 # Default false
 confidential_guest = true
 
-# Enable pre-attestation AMD SEV and SEV-ES confidential guests.
+# Enable pre-attestation for AMD SEV and SEV-ES guests.
 # Applies only if confidential_guest is true.
 # (default: false)
 guest_pre_attestation = true
 
-# Guest owner proxy that handles remote attestation
-guest_pre_attestation_proxy = "@DEFGUESTPREATTESTATIONPROXY@"
+# A remote key broker server (KBS) is required to validate the attestation
+# measurement and inject a secret key.  
+# See https://github.com/confidential-containers/simple-kbs
+guest_pre_attestation_kbs_proxy = "@DEFGUESTPREATTESTATIONPROXY@"
```

Comment:
I'm not sure about the term `proxy` here. This is the URI of the KBS. Where does a proxy come in?

Reply:
Agreed